packages/itwinui-react/src/core/Tree/TreeNode.tsx

```diff
@@ -238,7 +258,10 @@ export const TreeNode = (props: TreeNodeProps) => {
           onClick={() => !isDisabled && onSelected?.(nodeId, !isSelected)}
         >
           {checkbox && (
-            <Box className='iui-tree-node-checkbox'>
+            <Box
+              className={cx('iui-tree-node-checkbox', checkboxProps?.className)}
```

Comment:
The merged `className` should come after `{...checkboxProps}`, otherwise it will be overridden

Reply:
switched those in my last push

Reply:
It's only `className` that should come after. The other props should be before. 

Specifically:

```diff
  <Box
+   as='span'
+   aria-hidden
    {...iconProps}
-   as='span'
    className={cx(
      'iui-tree-node-content-icon',
      iconProps?.className,
    )}
-   aria-hidden
  >
```